lib/src/component_declaration/flux_component.dart

```diff
@@ -27,9 +27,62 @@ import 'annotations.dart' as annotations;
 import 'builder_helpers.dart';
 import 'component_base_2.dart';
 
-/// Builds on top of [UiProps], adding typed props for [Action]s and [Store]s in order to integrate with w_flux.
+/// Builds on top of [UiProps], adding typed [Action] and [Store] props for use with [FluxUiComponent2].
 ///
-/// Use with the over_react builder via the `@Props()` ([annotations.Props]) annotation.
+/// __Example:__
+///
+/// ```dart
+/// class YourComponentProps extends UiProps with FluxUiPropsMixin<YourFluxActionsClass, YourFluxStoreClass> {
+///   // Additional arbitrary props for your component can go here
```

Comment:
In the new boilerplate, this isn't true

Reply:
Ah... I was wondering about that... will have to update my codemod to reflect this.

Reply:
👍 I think I left some comments on that PR to that effect